src/templates/vault_transit.hcl.j2

```diff
@@ -0,0 +1,10 @@
+seal "transit" {
+  address         = "{{ vault_addr }}"
+  disable_renewal = "false"
```

Comment:
This made me think
- Will the token renew?
- The token expires right?
- If Vault B restarts after token ttl, will it come up?
- Are renewals supported by our implementatin?